src/tag.tsx

```diff
@@ -39,20 +39,19 @@ const Tag: React.FC<TagProps> = ({Icon, children, dataAttributes, type = 'promo'
     }
 
     const tagTypeToColors = {
-        // [textColor, inverseTextColor, backgroundColor]
-        promo: [colors.promoHigh, colors.promoHighInverse, colors.promoLow],
-        active: [colors.brand, colors.brand, colors.brandLow],
+        // [textColor, backgroundColor]
+        promo: [colors.tagTextPromo, colors.tagBackgroundPromo],
+        active: [colors.tagTextActive, colors.tagBackgroundActive],
         inactive: [
-            colors.neutralMedium,
-            colors.neutralMediumInverse,
-            themeVariant === 'alternative' ? colors.neutralLowAlternative : colors.neutralLow,
+            colors.tagTextInactive,
+            themeVariant === 'alternative' ? colors.neutralLowAlternative : colors.tagBackgroundInactive,
```

Comment:
this logic will be removed when Movistar skin is improved in terms of accessibility contrast. Just keep it to avoid visual changes in production

Reply:
Is there a Jira's issue for this? Maybe you can add it as a comment in the code